build/Common.props

```diff
@@ -22,17 +22,18 @@
       Please sort alphabetically.
       Refer to https://docs.microsoft.com/en-us/nuget/concepts/package-versioning for semver syntax.
     -->
+    <InMemoryExporterLatestStable>1.3.0</InMemoryExporterLatestStable>
```

Comment:
Consider `OpenTelemetryExporterInMemoryPkgVer`, the semantic here is "the version of in-memory exporter used in this project", not necessarily "the latest stable version" (think about this, if in-memory exporter shipped a new stable version, the semantic would be broken here).

Reply:
Yeah, we would need to update this when a newer stable version is released, and the packages are ready to onboard to it but if we need the packages to use a newer pre-release version, then the semantic meaning would break.